examples/scpclient/scpclient.c

```diff
@@ -313,7 +313,9 @@ THREAD_RETURN WOLFSSH_THREAD scp_client(void* args)
 
     ret = wolfSSH_shutdown(ssh);
     /* do not continue on with shutdown process if peer already disconnected */
-    if (ret != WS_SOCKET_ERROR_E && wolfSSH_get_error(ssh) != WS_SOCKET_ERROR_E) {
+    if (ret != WS_CHANNEL_CLOSED && ret != WS_SOCKET_ERROR_E &&
```

Comment:
Should wolfSSH_shutdown be returning WS_CHANNEL_CLOSED instead of WS_SUCCESS when the channel is closed?

Reply:
Channels first report they have reached End Of File first. After both ends of the channel EOF, they close.

As far as what should wolfSSH_shutdown return? I'm not sure yet. I need to get that Changing Channels PR in. The shutdown process will make more sense with the callbacks.

Reply:
Updated to return WS_CHANNEL_CLOSED